build/Dockerfile

```diff
@@ -7,8 +7,8 @@ ARG DATE
 
 COPY / /golangci
 WORKDIR /golangci
-RUN APP_VERSION=${VERSION#v} \
-    CGO_ENABLED=0 \
+RUN APP_VERSION=${VERSION#v}; \
```

Comment:
@ashmckenzie just for interest – how it works? :D

Reply:
The previous logic never worked as intended as passing variables to commands like this just doesn't work, for example:

```sh
$ VAR="test" echo "VAR=[${VAR}]"
VAR=[]

$ VAR="test"; echo "VAR=[${VAR}]"
VAR=[test]
```

Reply:
this is `sh` magic?
I remember the common pattern `FOO=bar cmd args` (or using `export`)

Reply:
This is more how the variable is scoped to `echo` only and not `sh`. 

```sh
› VAR="test" echo "VAR=[${VAR}]"
VAR=[]

› VAR="test" sh -c 'echo "VAR=[${VAR}]"'
VAR=[test]
```
In the first example `$VAR` is only visible to `echo` (not the assignment) but in the second example it's scoeped to `sh` so it will be in the scope for everything `sh` calls.

So in this case `APP_VERSION` is visible to `go` but not to the command line flags. Here's an example where you can read it as an env but not use it as an arg:

```sh
› VAR=test perl -E 'say $ENV{"VAR"}; say @ARGV' some-arg $VAR
test
some-arg
```

If you use `export` you make it globally visible so that's not the same.